Core/Executor/ContentTypeManager.php

```diff
@@ -553,9 +567,15 @@ protected function createFieldDefinition(ContentTypeService $contentTypeService,
                 case 'name':
                     $fieldDefinition->names = array($lang => $value);
                     break;
+                case 'names':
+                    $fieldDefinition->names = $value;
+                    break;
                 case 'description':
                     $fieldDefinition->descriptions = array($lang => $value);
                     break;
+                case 'descriptions':
+                    $fieldDefinition->description = $value;
```

Comment:
this should be `descriptions` with an `s`, right?

Reply:
You are right. I changed it.